Added the MapRemoteFloret (#220)
cauliframework/cauli: CHANGELOG.md

# Changelog

## Unreleased

> * **feature** Added a `MapRemoteFloret` to simply change urls of requests before they are sent. [#216](https://github.com/cauliframework/cauli/issues/216) by @brototyp

Reviewer: ```suggestion
* **feature** Added a `MapRemoteFloret` that can change urls of requests before they are sent. [#216](https://github.com/cauliframework/cauli/issues/216) by @brototyp
```
